wpdemo/bundle/aws/eks/env.yml

```diff
@@ -7,7 +7,7 @@ environment:
     driver: github
     repo: https://github.com/cldcvr/codepipes-tutorials
     dir: wpdemo/infra/aws/eks
-    identifier: main
+    identifier: rajat/VAN-2821
```

Comment:
should be removed before merge?

Reply:
yes we can @dshelley66 but the pipeline will fail with 1 test. Should we run the pipeline or can ignore it for this time?

Reply:
@dshelley66 @kanaksinghal @shwetanshu-cldcvr I have incorporated the review comments and the pipeline is failing with 1 test as expected.

[Result Link](https://storage.cloud.google.com/vg-api-test-report/automation/api/results/13-Jul-2022/14:23:08)

Reply:
do we have a fix queued up for the failing test? What is the jira for it?

Reply:
ah ok